### ISSUE TITLE
Dependency pointing to personal environment?

### ISSUE THREAD

**Issue by julianrojas87** [rdf-connect/orchestrator#31]
The `rdfc` dependency of the python SHACL validator is pointing to a local file with an absolute path:
https://github.com/rdf-connect/orchestrator/blob/41cd4039e5716541ab3141c3fb307ef01645e387/packages/shacl-validator-py/requirements.txt#L33

Shouldn't this be a relative path instead? Or shall we publish that dependency as a proper library?

**Comment by jenspots** [comment]
You're right, the `requirements.txt` is actually outdated and should be removed. The actual dependencies are listed in the `pyproject.toml` file.

https://github.com/rdf-connect/orchestrator/blob/41cd4039e5716541ab3141c3fb307ef01645e387/packages/shacl-validator-py/pyproject.toml#L11

However, the library itself should at one point be published separately. GitHub doesn't support publishing Python packages yet so I'll look into something like `pypi`.
